pallets/treasury-buyout-extension/src/lib.rs

```diff
@@ -241,8 +247,8 @@ impl<T: Config> Pallet<T> {
 		let (basic_asset_price, exchange_asset_price) = Self::fetch_prices((&basic_asset, &asset))?;
 
 		// Add fee to the basic asset price
-		let basic_asset_price_with_fee =
-			basic_asset_price * (FixedU128::from(T::SellFee::get()) + FixedU128::one());
+		let fee_plus_one = FixedU128::from(T::SellFee::get()).checked_add(&FixedU128::one()).expect("This should never fail");
```

Comment:
Nit: Yes it should never fail. But let's not use `unwrap()` or `expect()` in any production code of our pallets. Tests and benchmarks can be treated differently. But if for some reason, the `SellFee` is the maximum possible value and we want to add `1` extra, this would make our runtime panic and halt our chain.
Even if there is no such scenario, we should still not use `unwrap()`/`expect()` anywhere. Instead please make this return a custom error instead. I'm fine if we have a generic name like `MathError` for this.

Reply:
Completely agree! Going to change this now